src/main/java/seedu/workbook/ui/HelpWindow.java

```diff
@@ -15,9 +15,9 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String ADD_COMMAND = "add c/COMPANY r/ROLE s/STAGE [d/DD-MMM-YYYY] [e/EMAIL] "
+    public static final String ADD_COMMAND = "add c/COMPANY r/ROLE s/STAGE [d/DATE] [e/EMAIL] "
```

Comment:
Should we just standardise everywhere (at least user facing incl. UG/DG) to use DATETIME since we do require time for this parameter right

Reply:
Yup sure gd point.